projects/assets/src/translations/translation-chunks-config.ts

```diff
@@ -48,6 +48,7 @@ export const translationChunksConfig: TranslationChunksConfig = {
     'updatePasswordForm',
     'updateProfileForm',
     'consentManagementForm',
+    'newConsentManagement',
```

Comment:
`newConsentManagement ` is only for new-myaccount. It's dedicated for CDC/CDP.  And for now, it won't replicated consentManagement. Only after CDC/CDP changing cms config and introduce `newConsentManagement', it will be displayed on SPA UI. That's why there is no e2e test. But we already have a ticket to track it.
https://jira.tools.sap/browse/CXSPA-4498